cmd/xl-v1-object.go

```diff
@@ -266,7 +267,7 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 
 		// Get the checksums of the current part.
 		checkSums := make([]string, len(onlineDisks))
-		var ckSumAlgo HashAlgo
+		var ckSumAlgo bitrot.Algorithm = 0xffffffff
 		for index, disk := range onlineDisks {
```

Comment:
Any reason why choosing `0xffff` to indicate a default value . Can we use something else ?